Lesson_2_Python_II/Lesson_2_clean.ipynb

```diff
@@ -6,49 +6,34 @@
    "source": [
```

Comment:
<p>-Did you mean to delete the line that reads 'plot function tells Python what to plot an dhow to plot it'?</p><p>-decades*</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theLeague-AI/TheLeagueAI_main/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>yeah I meant to delete it, for now no need to have this function explained</p><p> </p>

Reply:
<p>Will add it back, it does make the content more clear</p><p> </p>